### PR TITLE
Remove some redundant overrides

### PR DESCRIPTION
**Pull request by adleong**
grpc-gateway  is already explicitly on v1.16.0 and containerd is already explicitly on v1.6.1

Signed-off-by: Alex Leong <alex@buoyant.io>

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -540,12 +650,16 @@ github.com/grpc-ecosystem/go-grpc-middleware v1.0.1-0.20190118093823-f849b5445de
 github.com/grpc-ecosystem/go-grpc-middleware v1.3.0/go.mod h1:z0ButlSOZa5vEBq9m2m2hlwIgKw+rp3sdCBRoJY+30Y=
 github.com/grpc-ecosystem/go-grpc-prometheus v1.2.0 h1:Ovs26xHkKqVztRpIrF/92BcuyuQ/YW4NSIpoGtfXNho=
 github.com/grpc-ecosystem/go-grpc-prometheus v1.2.0/go.mod h1:8NvIoxWQoOIhqOTXgfV/d3M/q6VIi02HzZEHgUlZvzk=
-github.com/grpc-ecosystem/grpc-gateway v1.14.8 h1:hXClj+iFpmLM8i3lkO6i4Psli4P2qObQuQReiII26U8=
-github.com/grpc-ecosystem/grpc-gateway v1.14.8/go.mod h1:NZE8t6vs6TnwLL/ITkaK8W3ecMLGAbh2jXTclvpiwYo=
+github.com/grpc-ecosystem/grpc-gateway v1.9.0/go.mod h1:vNeuVxBJEsws4ogUvrchl83t/GYV9WGTSLVdBhOQFDY=
+github.com/grpc-ecosystem/grpc-gateway v1.9.5/go.mod h1:vNeuVxBJEsws4ogUvrchl83t/GYV9WGTSLVdBhOQFDY=
```
Comment (olix0r) [inline go.sum:654]: note that this introduces older grpc-gateway dependencies. i'm not sure if this is a problem...

File: go.sum [hunk 2]
```diff
@@ -198,47 +223,122 @@ github.com/cockroachdb/datadriven v0.0.0-20190809214429-80d97fb3cbaa/go.mod h1:z
...
+github.com/containerd/containerd v1.4.1/go.mod h1:bC6axHOhabU15QhwfG7w5PipXdVtMXFTttgp+kVtyUA=
+github.com/containerd/containerd v1.4.3/go.mod h1:bC6axHOhabU15QhwfG7w5PipXdVtMXFTttgp+kVtyUA=
+github.com/containerd/containerd v1.4.9/go.mod h1:bC6axHOhabU15QhwfG7w5PipXdVtMXFTttgp+kVtyUA=
+github.com/containerd/containerd v1.5.0-beta.1/go.mod h1:5HfvG1V2FsKesEGQ17k5/T7V960Tmcumvqn8Mc+pCYQ=
+github.com/containerd/containerd v1.5.0-beta.3/go.mod h1:/wr9AVtEM7x9c+n0+stptlo/uBBoBORwEx6ardVcmKU=
+github.com/containerd/containerd v1.5.0-beta.4/go.mod h1:GmdgZd2zA2GYIBZ0w09ZvgqEq8EfBp/m3lcVZIvPHhI=
+github.com/containerd/containerd v1.5.0-rc.0/go.mod h1:V/IXoMqNGgBlabz3tHD2TWDoTJseu1FGOKuoA4nNb2s=
+github.com/containerd/containerd v1.5.1/go.mod h1:0DOxVqwDy2iZvrZp2JUx/E+hS0UNTVn7dJnIOwtYR4g=
+github.com/containerd/containerd v1.5.7/go.mod h1:gyvv6+ugqY25TiXxcZC3L5yOeYgEw0QMhscqVp1AR9c=
+github.com/containerd/containerd v1.5.8/go.mod h1:YdFSv5bTFLpG2HIYmfqDpSYYTDX+mc5qtSuYx1YUb/s=
```
Comment (olix0r) [inline go.sum:263]: Similarly, we now depend on many older containerd versions. This seems potentially problematic, as these have security vulnerabilities.

**Review by olix0r — COMMENTED** [review]
I'm not really sure how all of this works, but it seems like this could be a problem if it causes us to pull in dependencies that trigger security vuln scanners

File: go.sum (hunk 1 above)
Comment (adleong) [reply go.sum:544]: I'm not really sure how this works either.  I guess this means that we have a transitive dependency on grpc-gateway 1.9.0, for example?  I suppose this means we should not remove the `replace` directive until all of our dependencies have upgraded to 1.14.8 or later.

**Review by adleong — COMMENTED** [review]
Exactly how these replace directives (and dependency management in general) works is still a mystery to me and it's very hard to find useful documentation that actually describes how this is supposed to work.  I think we probably need to keep these `replace` directives around until the offending versions no longer appear anywhere in our transitive dependency tree (although it's not clear how to track this other than by periodically removing the `replace` and seeing what shows up in `go.sum`).

**Pull request closed without merging**